### PR TITLE
fix: individual post view added

### PR DESCRIPTION
**Pull request by ani2607**
## Related Issue
#121 

## Description

I just created a route of <base_url>/post/postId  which substantially created a page for rendering the individual post with the specific post id.

## Screenshots

image
image

### CODE REVIEW

File: src/components/core/post/index.tsx
Comment (Sanchitbajaj02) [inline src/components/core/post/index.tsx:1]: Couldn't the Singlepost.tsx be used there?

**Review by Sanchitbajaj02 — CHANGES_REQUESTED** [review]

**Comment by Sanchitbajaj02** [comment]
@ani2607 PR is reviewed. Please also remove the merge conflicts as well

File: src/components/core/post/index.tsx
Comment (ani2607) [reply src/components/core/post/index.tsx:1]: Yaa! would update.

Comment (ani2607) [reply src/components/core/post/index.tsx:1]: Yaa! Updated , please check

**Review by Sanchitbajaj02 — APPROVED** [review]

**Review by Sanchitbajaj02 — APPROVED** [review]

**Comment by Sanchitbajaj02** [comment]
@ani2607 PR reviewed. Please resolve the merge conflicts of yarn.lock.
To fix this issue, delete the current yarn.lock and generate new one and then push

**Review by Sanchitbajaj02 — APPROVED** [review]

**Pull request merged**